compiler/damlc/tests/daml-test-files/SemanticsValue.daml

```diff
@@ -24,26 +24,28 @@
 -- ValExpTyAbsNat case.
 module SemanticsValue where
 
-main = scenario do
-    let _ : Int -> Int = (\ (x : Int) -> error "ValExpAbs")
-    -- ValExpTyAbsNat is tested in SemanticsValueSince17
-    let _ : Update Int = do
-        _ <- pure ()
-        error "ValUpdateBind"
-    p <- getParty "Alice"
-    let _ : Scenario Int = submit p do
-        error "ValUpdateEmbedExpr (1)"
-    let _ : Scenario () = submitMustFail p do
-        error "ValUpdateEmbedExpr (2)"
-        -- ^ damlc uses 'embed_expr' when compiling a submit
-        -- or a submitMustFail ... however, this may interact
-        -- with the semantics of 'commit', so we should also
-        -- implement a more direct test in the LF interpreter.
-    let _ : Scenario () = do
-        _ <- pure ()
-        error "ValScenarioBind"
-    -- cannot test ValScenarioEmbedExpr from Daml, since
-    -- the compiler does not generate it.
-    pure ()
+-- @TODO Similar to SubmitFail, Script has different submission semantics, breaking this test
+
+import Daml.Script
 
--- @ENABLE-SCENARIOS
+main = script do
```

Comment:
I would have keep the scenario version, until we find a solution.

Reply:
Have done, though I'm not sure there is really a solution to find. The semantics have changed, either we remove the test, or change it to look for different semantics